components/VcDetails.tsx

```diff
@@ -68,10 +70,28 @@ export const VcDetails: React.FC<VcDetailsProps> = props => {
             </Column>
             <Row>
               <Column>
-                <Column>
+                <Column margin="20 0 0 0">
+                  <Text
+                    testID="gender"
+                    weight="regular"
+                    size="smaller"
+                    color={Theme.Colors.DetailsLabel}>
+                    {t('gender')}
+                  </Text>
+                  <Text
+                    testID="genderValue"
+                    weight="semibold"
+                    size="smaller"
+                    color={Theme.Colors.Details}>
+                    {getLocalizedField(
+                      props.vc?.verifiableCredential.credentialSubject.gender,
+                    )}
+                  </Text>
+                </Column>
+                <Column margin="25 0 0 0">
                   <Text
                     testID="idType"
-                    weight="bold"
+                    weight="regular"
```

Comment:
Did you check with "semi-bold"? Because regular might differ from VcDetaile text in the wireframe.

Reply:
The style mentioned int the wireframe is regular